### PR TITLE
#52: Resolved Interactive book bug

### PR DESCRIPTION
**Pull request by kileha3**
Related to #52

- Removed babel babel-polyfill error by removing duplicated JS script dependencies (h5p-interactive-book.js was included twice hence causing it to be run twice)
- Resolved undefined title error on h5p-interactive-book.js when loaded by adding metadata on H5PIntegration.contents (Was missing)
- Added test case for interactive book scenario

### CODE REVIEW

**Comment by fkohrt** [comment]
I can confirm this works for me.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "h5p-standalone",
```
Comment (zbynek) [inline package.json:2]: This seems to be useful for testing purposes, but should probably not be included in the PR.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 node_modules
 .DS_Store
-cypress/videos
\ No newline at end of file
+cypress/videos
+dist
```
Comment (zbynek) [inline .gitignore:4]: I think `dist` is included in the repository on purpose so that people can download the library and use it without npm.

Comment (fkohrt) [reply .gitignore:4]: An alternative would be uploading the content of `dist/` as a release in GitHub.

File: src/js/h5p-standalone.class.js [hunk 3]
```diff
@@ -18,14 +18,13 @@ export default class H5PStandalone {
   constructor(el, pathToContent = 'workspace', options = {}, displayOptions = {}, librariesPath) {
     this.id = options.id || Math.random().toString(36).substr(2, 9);
     this.path = pathToContent;
+    this.H5P = H5P;
```
Comment (zbynek) [inline src/js/h5p-standalone.class.js:21]: What's the point of this assignment? Will it just make `this.H5P` point to `window.H5P` or am I missing something?

File: vendor/h5p/js/h5p.js [hunk 4]
```diff
@@ -218,7 +218,7 @@ H5P.init = function (target) {
 
     // Listen for xAPI events.
     H5P.on(instance, 'xAPI', H5P.xAPICompletedListener);
-
+
```
Comment (zbynek) [inline vendor/h5p/js/h5p.js:221]: Whitespace change can be reverted

File: src/js/h5p-standalone.class.js [hunk 5]
```diff
@@ -211,10 +211,18 @@ export default class H5PStandalone {
     });
 
     if (this.mainLibrary.preloadedCss) {
-      Array.prototype.push.apply(styles, this.mainLibrary.preloadedCss.map(style => `${this.librariesPath}/${this.mainLibraryPath}/${style.path}`));
+      $.each(this.mainLibrary.preloadedCss, (i, style) => {
```
Comment (zbynek) [inline src/js/h5p-standalone.class.js:214]: just realized this doubles the file size of the resulting bundle because of extra copy of jQuery. It would be better to use `forEach` and `indexOf` and not depend on jQuery.

**Comment by kileha3** [comment]
Hey @zbynek , thanks for the review I'll definitely work on them after merging the latest changes from primary.

File: src/js/h5p-standalone.class.js (hunk 3 above)
Comment (kileha3) [reply src/js/h5p-standalone.class.js:21]: The idea behind this was to expose the h5p instance to the outside world so that somebody can use the same instance for other purposes. E.g xAPI Statements.
